recipes/intake-stac/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "intake-stac" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
@ocefpaf - again, I'm having trouble with this URL scheme. Ideas?

https://pypi.org/project/intake-stac/

Reply:
PyPI mangles `-` and `_` in a very confusing way. In some places they are interchangeable, in other they are not.